youtube_dl/extractor/hrti.py

```diff
@@ -129,6 +129,22 @@ class HRTiIE(HRTiBaseIE):
     }, {
         'url': 'hrti:2181385',
         'only_matching': True,
+    }, {
+        'url': 'https://hrti.hrt.hr/video/show/3873068/cuvar-dvorca-dramska-serija-14',
```

Comment:
`only_matching`.

Reply:
Remove the entire `info_dict` and add `'only_matching': True,` or just add `'only_matching': True,` and keep the `info_dict`.

Reply:
`info_dict` is pointless with `only_matching`.